.github/workflow/doppler-firebase-deploy.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will install Doppler CLI and Firebase tools, then deploy to Firebase
+
+name: Firebase with Doppler CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
```

Comment:
Do we want it to also deploy anytime a pull request to master is opened? I think just push to master is sufficient.